Game/player.gd

```diff
@@ -13,16 +13,27 @@ var can_jump = false
 @export var wall_jump_power = 500
 
 @onready var test_animation = $AnimatedSprite2D
-
+var initial_position : Vector2
+	
 func _ready():
 	test_animation.play("idle animation")
+	initial_position = position
 
 func _on_coyote_timer_timeout():
 	can_jump = false
 
 var air_time := 0.0
 
 func _physics_process(delta):
+	if (position.y > 1092.31863281):
```

Comment:
I think this is a slightly more reliable way of checking that the player fell off the bottom.
```suggestion
	if (position.y > get_viewport_rect().end.y):
```